.gitignore

```
@@ -22,6 +22,7 @@ coverage.xml
 /build
 /docs/_build
 /testing/ops_scenario.egg-info/
+/testing/src/ops_scenario.egg-info/
```

WDYT about making this simpler?

```suggestion
ops_scenario.egg-info
```